### PR TITLE
Use augur merge

### LINKED ISSUE #73: Use `augur merge` to spike in USVI records
### Description

Use the new `augur merge` feature to spike in USVI records into the phylogenetic build

### Possible solution

* [x] Add URL column to [USVI records](https://github.com/nextstrain/zika/blob/main/phylogenetic/data/metadata_usvi.tsv) and [example USVI record](https://github.com/nextstrain/zika/blob/main/phylogenetic/example_data/metadata_usvi.tsv)
* [x] Replace [csvtk merging](https://github.com/nextstrain/zika/blob/649e495a6b181c004bacae26ea4507fc6989abc5/phylogenetic/rules/merge_sequences_usvi.smk#L48-L53) with `augur merge`

### PR DESCRIPTION
**Pull request by victorlin**
## Description of proposed changes

Use our own tooling designed for this type of operation. Details in commits.

## Related issue(s)

- Closes #73

## Checklist

- [x] Tested locally. Differences noted in commit messages.
- [x] Checks pass

### CODE REVIEW

**Review by j23414 — APPROVED** [review]
Agree with your changes and ran a quick test:

* https://next.nextstrain.org/staging/zika/trials/victorAugurMerge/zika?f_country=USVI

Findings: USVI records were merged in as expected, node callout links work as expected, the difference in quoting does not seem to affect the auspice view. Good, thank you!

File: phylogenetic/rules/merge_sequences_usvi.smk [hunk 1]
```diff
@@ -21,34 +21,46 @@ This part of the workflow usually includes the following steps:
...
+        metadata = "data/metadata_modified.tsv"
     shell:
         """
-        cat {input.sequences} {input.usvi_sequences} > {output.sequences}
-
         csvtk mutate2 -tl \
           -n url \
           -e '"https://www.ncbi.nlm.nih.gov/nuccore/" + $genbank_accession' \
           {input.metadata} \
         | csvtk mutate2 -tl \
```
[the remark this answers is not in the document]
Comment (j23414) [reply phylogenetic/rules/merge_sequences_usvi.smk:42]: I ran a quick check that dropping the `-l` option wouldn't trigger the issues: https://github.com/nextstrain/zika/pull/58 or https://github.com/nextstrain/pathogen-repo-guide/pull/22

I agree that we can drop `-l`. Can either do that here or leave it as part of handling issue https://github.com/nextstrain/zika/issues/77

[the remark this answers is not in the document]
Comment (genehack) [reply phylogenetic/rules/merge_sequences_usvi.smk:42]: …or #74 :grin:

[the remark this answers is not in the document]
Comment (victorlin) [reply phylogenetic/rules/merge_sequences_usvi.smk:42]: Thanks, I'll leave it as-is in this PR

**Pull request merged**